crypto/envelope.go

```diff
@@ -0,0 +1,191 @@
+package crypto
+
+import (
```

Comment:
A small thing, but as a convention, we usually group our imports -- stdlib imports go first, unnamed imports then, and named imports last.